ruby/tests/well_known_types_test.rb

```diff
@@ -13,10 +13,15 @@ def test_timestamp
     assert_equal Time.at(12345), ts.to_time
     assert_equal 12345, ts.to_i
 
-    ts.from_time(Time.at(123456, 654321))
+    time = Time.at(123456, 654321)
+    ts.from_time(time)
     assert_equal 123456, ts.seconds
     assert_equal 654321000, ts.nanos
-    assert_equal Time.at(123456.654321), ts.to_time
+    assert_equal time, ts.to_time
+
+    time = Time.now
```

Comment:
I'd rather not make the test depend on the current time. We should make it deterministic. If it's important to use a time other than the `123456`/`654321` one, let's hard-code a time that exercises what we are testing.